src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.Designer.cs

```diff
@@ -11547,8 +11547,8 @@ internal class NuGetCommand {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to nuget sign MyPackage.nupkg -CertificatePath /path/to/certificate
-        ///nuget sign MyPackage.nupkg -CertificateFingerprint certificate_fingerprint -OutputDirectory ./signed/.
+        ///   Looks up a localized string similar to nuget sign MyPackage.nupkg -CertificatePath \path\to\certificate
```

Comment:
Look at other command examples (e.g.:  `nuget add`) and copy them.  For example:

    nuget sign MyPackage.nupkg -CertificatePath C:\certificate.pfx